stock_picking_batch_back2draft/views/stock_picking_batch_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_picking_batch_form_reopen" model="ir.ui.view">
+        <field name="name">view_picking_batch_form_reopen</field>
+        <field name="model">stock.picking.batch</field>
+        <field name="inherit_id" ref="stock_picking_batch.stock_picking_batch_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='cancel_picking']" position="after">
+                <button name="action_back_to_draft" states="cancel" string="Back to draft" groups="base.group_user" type="object"/>
```

Comment:
Maybe a help to explain a little bit more.

Reply:
Done